public/locales/ua/common.json

```diff
@@ -25,7 +25,6 @@
   "products": "Харчування",
   "services": "Послуги",
   "others": "Інші",
-  "volunteer": "Волонтер",
```

Comment:
keep with the respective translation for Volunteering
and check signup.volunteering.header to be the same